tests/test_utils.py

```diff
@@ -23,23 +23,28 @@ def setUp(self):
     def tearDown(self):
         supported_models.cache_clear()
 
-    def test_supported_models(self):
-        models = ["Foo", "Bar"]
-        app_config = Mock(
-            spec=[], cms_extension=Mock(spec=[], navigation_apps_models=models)
-        )
-        with patch.object(apps, "get_app_config", return_value=app_config):
-            self.assertEqual(supported_models(), models)
+    def test_supported_models_with_installed_apps(self):
+        """Tests that all models registered in cms_config shows
+        up in supported models"""
+
+        registered_models = supported_models()
+
+        expected_models = {
+            TestModel1: [],
+            TestModel2: [],
+            TestModel3: [],
+            TestModel4: [],
+            Page: ["title"],
+            PollContent: ["text"],
+        }
+        self.assertDictEqual(registered_models, expected_models)
 
     @patch.object(apps, "get_app_config", side_effect=LookupError)
     def test_supported_models_returns_empty_list_on_lookup_error(self, mocked_apps):
         self.assertDictEqual(supported_models(), {})
 
     def test_supported_models_is_cached(self):
-        models = ["Foo", "Bar"]
```

Comment:
This looks liek something Monika would have done. If it was created by Monika there is a reason that she's mocking here. I won't reject this change but try and understand why she may have mocked here.

Reply:
Using the app_config as a mock object will get you an error. Generally I'm of the opinion if you can use real objects you should as that will help you avoid bugs. For this particular example I can't find a reason to use a mock for the app_config.  

In this case we're testing the cache and with the approach below it absolutely fulfills that criteria.